### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by Krinkle**

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,42 +1,66 @@
-# grunt-banana-checker Release History
+v0.7.0 / 2019-01-08
+==================
+
+* Add check for message key case validity (James D. Forrester)
+* Add option to require a message key prefix (James D. Forrester)
+* build: Replace jshint and jscs with eslint (James Forrester)
```
Comment (jdforrester) [inline History.md]: Inconsistent.

Comment (Krinkle) [reply History.md]: Aye, that's from git-log. Will fix.

Comment (Krinkle) [reply History.md]: Also, today I learned Git doesn't use mailmap for git-log by default, only for git-shortlog. But, it is [configurable](https://stackoverflow.com/a/30900151/319266).

**Comment by Krinkle** [comment]
Rebased to rebuild lock file for the eslint-config-wikimedia bump.

**Comment by jdforrester** [comment]
Will you do the mechanics of the release (signed tag, npm publish) or should I? Should we do a wikitech-l post?

**Comment by Krinkle** [comment]
@jdforrester Done (signed git, npm). For Wikitech-l, I'm not sure it's worth announcing at this point. Might be more valuable for the next release?

**Pull request merged**